### PR TITLE
6 api get comments by article

### PR DESCRIPTION
**Pull request by JenWarman**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -80,32 +82,76 @@ describe('nc-news API', () => {
...
+                })
+        })
+        test('400: responds with error message if article_id is invalid', () => {
+            return request(app)
+                .get('/api/articles/IamNotAnId')
+                .expect(400)
+                .then(({ body }) => {
+                    expect(body.msg).toBe('Bad request');
+                })
+        })
```
Comment (nc-kirsty-butterworth) [inline __tests__/app.test.js:155]: Really great test suite. Another one you could add here is 
Status 200 - responds with an empty array when the id is valid but there are no comments

File: models/topics.models.js [hunk 2]
```diff
@@ -52,5 +52,10 @@ exports.fetchArticles = () => {
     });
 }
 
-
+exports.fetchComments = (article_id) => {
+    return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`, [article_id])
+        .then(({ rows }) => {
+            return rows;
+        })
```
Comment (nc-kirsty-butterworth) [inline models/topics.models.js:59]: Once you add the additional test i have mentioned you are going to need to check the reason for the rows being returned empty. Is it because the article exists with no comments or because the article itself doesn't exist.
